src/components/downtime-counter/index.ts

```diff
@@ -30,18 +30,20 @@ type DowntimeCounter = {
   alert: ProbeAlert
   probeID: string
   url: string
+  createdAt?: Date
 }
 
 export function startDowntimeCounter({
   alert,
+  createdAt,
   probeID,
   url,
 }: DowntimeCounter): void {
   const newIncident = {
     alert,
     probeID,
     probeRequestURL: url,
-    createdAt: new Date(),
+    createdAt: createdAt || new Date(),
   }
 
```

Comment:
**Suggestion:** Consider using default parameters instead of the logical OR operator to set default values.
```suggestion
export function startDowntimeCounter({
  alert,
  createdAt = new Date(),
  probeID,
  url,
}: DowntimeCounter): void {
  const newIncident = {
    alert,
    probeID,
    probeRequestURL: url,
    createdAt,
  }

  setContext({ incidents: [...getContext().incidents, newIncident] })
}
```